constraints.txt

```diff
@@ -0,0 +1,2 @@
+numba>=0.60.0
+numpy<2.0
```

Comment:
I’m pretty sure that uv supports comments by having lines start with `#`.

Could you please comment on why these constraints exist?

I assume the numba is because of resolver backtracking and numpy because one core package or another doesn’t support it yet but fails to specify that upper bound? If I’m right, please also add a link to the issue for numpy 2 compat in that project.